### DISCUSSION TITLE
package registry with auth code

### DISCUSSION THREAD

**Discussion by justforuse** [oven-sh/bun#5887] [Q&A]
I have check the docs about custom package registry: https://bun.sh/docs/install/registries

But for our Org's .npmrc, is look like this for now:
```
registry=https://my.company.com/npm/path/
//my.company.com/npm/path/:_auth=SECURECODE
```

I tried add this code to .bunfig.toml, but still output `package not found`
```
[install]
# set default registry as a string
registry = { url = "https://my.company.com/npm/path/", _auth = 'SECURECODE' }
```

How can I set it to `.bunfig.toml`, so I can use Bun as the package manager to install our packages?

Thanks

**Comment by melroy89** [comment]
Try to use `token` instead of `_auth`

**Comment by melroy89** [comment]
To correct myself, you can also try the `_auth` as follows within the URL: `url = "https://my.company.com/api/npm/npm/_auth=MY_TOKEN"`


You can also make it short by putting `registry` in the setting option in the toml config, like so:

```toml
[install.registry]
url = "https://my.company.com/api/npm/npm/_auth=MY_TOKEN"
```

**Reply by justforuse** [reply]
Thanks your answer. I tried these methods, but seems still not work 😢

**Reply by melroyvandenberg** [reply]
ow that is too bad. I also read something about proxy issues. Especially socks5. Do you also use proxies maybe?
